styles.css

```diff
@@ -0,0 +1,704 @@
+    width: 100%;
+    margin-bottom: 6.7vw;
+  }
+
+  #footer-line {
+    border: 5px solid #0d151e;
+    width: 35.8vw;
+    margin: 12.5vw auto 3vw auto;
+    border-radius: 2.7vw;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  /* Nav bar styling */
+  #navbar {
+    padding: 0 10.8vw 0 10.5vw;
+  }
+
+  #logo {
+    width: auto;
+    height: 5vw;
+    border-radius: 50%;
+  }
+
+  #menu-icon {
+    display: none;
+  }
+
+  .desk-nav {
+    margin-right: 2.2vw;
+    font-weight: 600;
+    font-size: 1vw;
+  }
+
+  #desk-nav {
+    display: flex;
+    align-items: center;
+  }
+
+  #home {
+    background-image: url(media/desktop-header.png);
+    background-size: 45%;
+    background-position: right top;
+    display: flex;
+    padding: 13.3vw 47vw 17.4vw 2.5vw;
+    align-items: center;
+  }
+
+  .orange-color {
+    font-size: 5vw;
+  }
+
+  .text {
+    font-weight: 400;
+    font-size: 1.4vw;
+    line-height: 2.2vw;
+  }
+
+  .orange-button {
+    background-color: #ff6b00;
+    padding: 1.2vw 0;
+    font-weight: 700;
+    font-size: 1.2vw;
+    color: #fff;
+    border: 0;
+  }
+
+  article p {
+    font-weight: 400;
+    margin: 1.7vw 0;
+  }
+
+  .project-item:hover .btn,
+  .project-item:hover p,
+  .project-item:hover h2 {
+    display: none;
+  }
+
+  #home h2 {
+    font-weight: 500;
+    font-size: 2.2vw;
+    color: #fff;
+    margin: 1.7vw 0;
+  }
+
+  /* Social media links styling */
+  #social-items {
+    margin-right: 6vw;
+  }
+
+  .social-item {
+    margin-top: 1.4vw;
+  }
+
+  #projects {
+    margin: 9.4vw 10vw 9vw 9.8vw;
+  }
+
+  #project-intro {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    align-content: center;
+  }
+
+  #work-title {
+    font-family: 'Crete Round', sans-serif;
+    width: 22vw;
+    margin-bottom: 8vw;
+  }
+
+  .project-title {
+    font-family: 'Crete Round', sans-serif;
+    font-size: 2.8vw;
+    font-weight: 400;
+  }
+
+  article {
+    width: 28vw;
+    float: right;
+    margin-bottom: 9.5vw;
+  }
+
+  .line {
+    background-color: #212121;
+    height: 0.035vw;
+    border: 0;
+    width: 54vw;
+    margin: 2vw 0 9.25vw 0;
+  }
+
+  #work-cover {
+    width: 49vw;
+    height: auto;
+    margin-bottom: 1.7vw;
+    float: left;
+  }
+
+  #project-items {
+    display: grid;
+    grid: repeat(2, 30vw)/repeat(3, 25.6vw);
+    grid-gap: 2.6vw 1.7vw;
+  }
+
+  .project-item {
+    display: flex;
+    flex-direction: column;
+    justify-content: end;
+    padding: 1.1vw;
+  }
+
+  .project-item .orange-button {
+    display: none;
+  }
+
+  .title {
+    font-size: 2.2vw;
+  }
+
+  .project-item .text {
+    color: #fff;
+    font-size: 1.15vw;
+    line-height: 1.7vw;
+    margin: 1.6vw 0;
+  }
+
+  .grey-button {
+    float: left;
+    background-color: rgba(255, 255, 255, 0.24);
+    color: #fff;
+    border: inherit;
+    font-weight: 500;
+    font-size: 1vw;
+    padding: 0.7vw 0.83vw;
+    margin-right: 0.83vw;
+  }
```

Comment:
I kindly suggest you turn the container of the buttons into a flex container so that you don't have hover issues.